src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -28,12 +31,41 @@ const HomeScreen = observer((_props: HomeScreenProps) => {
     });
   };
 
+  const getEvents = (_location?: string) => {
+    if (events.allEvents.length > 0) {
```

Comment:
I see that you usually add checks with return first. Instead we could write like this - 

```
if (!events.allEvents.length) {
  events.getAllEvents(_location);
}
```